### DISCUSSION TITLE
v8 Range selection with two inputs

### DISCUSSION THREAD

**Discussion by DylanAllen** [gpbl/react-day-picker#1427] [Support] [answered]
I am upgrading from v7 to v8 and I need a range selection with an input component for both that start and end date. Is there a way to do this with the `useInput` hook, or any other hook? Visual reference of what I am doing with v7 currently:

---
<img width="303" alt="Screen Shot 2022-04-04 at 12 14 44 PM" src="https://user-images.githubusercontent.com/4248884/161596756-57a591df-350f-4c5e-9086-6ca69e3165c9.png">
:

**Comment by gpbl** [comment, marked as answer]
@DylanAllen the `useInput` hook is just for a single input field. If you need something more elaborated, you need to implement your own.

Checkout this example for a range select with two input fields:
https://react-day-picker.js.org/guides/input-fields#example-range-selection

**Comment by McSam27** [comment]
Dylan, were you able to get a date range picker working with two inputs? I'm also working on the same concept.

My v7 implementation also looks very similar to yours.

**Reply by DylanAllen** [reply]
@McSam27 I did, but I have since switched companies, so I don't have access to that code anymore. Sorry I can't be of more help.

I do remember it was not exactly straight forward, and the interactions didn't match the v7 exactly, but I was able to get close.

**Reply by McSam27** [reply]
Makes sense, I'm working on it now. Thanks for the response.
